index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel='stylesheet' type='text/css' media='screen' href='styles.css'>
+</head>
+<body>
+    
+    <div class="wrapper">
+
+        <header class="header">
+            <ul>
+                <li class="welcome"><a><h1>Welcome</h1></a></li>
+                <li class="toggle-menu"><img alt="burger" src="img/Union.svg" /></li>
+            </ul>
+        </header>
+    
+    <section class="content">
+        <div class="p1">
+            <p class="name">Hey there. I'm Diego</p>
+            <p class="car">I'm a software developer</p>
+        </div>
+        <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </section>
+
+    <section class="footer">
+        
+            <img class="twitter-logo" alt="twitter-logo" src="img/twitter.svg" />
+            <img class="linkdn-logo" alt="Linkedin-logo" src="img/Linkedin_icon.svg" />
+            <img class="medium-logo" alt="medium-logo" src="img/medium.svg" />
+            <img class="github-logo" alt="github-logo" src="img/github.svg" />
+            <img class="angelist-logo" alt="angellist-logo" src="img/angellist.svg" />
+        
+    </section>
```

Comment:
- [ ] For semantic reasons, Please remove this list of social links from this footer `section` and add them to the section above then please wrap each icon in a `<a>` tag and add them inside an unordered list `<ul>` and `<li>` tags for example:
```html
<ul>
    <li>
       <a href="add link here"><img class="twitter-logo" alt="twitter-logo" src="img/twitter.svg" /></a>
    </li>
    ..............
    ..............
</ul>
```